packages/core/src/RenderingEngine/renderingEngineCache.ts

```diff
@@ -36,6 +36,18 @@ const renderingEngineCache = {
     const renderingEngineIds = Object.keys(cache);
     const renderingEngines = renderingEngineIds.map((id) => cache[id]);
 
+    // sort the rendering engines so that the ones that start with _
```

Comment:
Please add an explanation of why the _ ones are at the end, they are .... and applied as ... because ...

Reply:
I second this.

Reply:
Thanks for filling in the comment nicely. Perhaps also mention what the underscore signifies?